docs/development/release_guide.rst

```diff
@@ -69,22 +69,22 @@ Release
 
 * Commit and push your changes up until now.
 
-* Open them up as a Pull Request from the ``0.5.x`` branch to the master branch.
+* Open them up as a Pull Request from the ``0.6.x`` branch to the master branch.
 
 * Make sure that tests pass and that documentation builds without issue.
 
 * No, really, check twice.
 
-  * If you want to do any rebase to clean up the commit history on your ``0.5.x``
+  * If you want to do any rebase to clean up the commit history on your ``0.6.x``
     branch, now is the time to do that. Ensure that no tests broke.
 
 * Tag the new version with ``git tag -s v<version> -m "Version v<version>"``
 
   * Note that ``-s`` signs the commit with your GPG key.
 
 * Push the tagged commit to the version's branch on GitHub: ``git push --force
-  --follow-tags upstream v0.5.x``. Note that ``--force`` is necessary to trigger
-  a rebuild with the tagged version. This kicked us in the posterior for ``0.5.0``.
+  --follow-tags upstream v0.6.x``. Note that ``--force`` is necessary to trigger
+  a rebuild with the tagged version. This kicked us in the posterior for ``0.6.0``.
```

Comment:
It's probably unnecessary to bump all of these up for each release - and this line was not even historically accurate in the first place, we had issues with the tagged version not being rebuilt in 0.4.0 :smile:

Reply:
Ah, oops!  I've had enough off-by-one errors in my life so I figured it'd be worth the 20 seconds to find and replace since I was in the area anyway, but this led to an off-by-one error that became an off-by-two error.  Thank you for finding this!